src/frontend/src/observer/observer_manager.rs

```diff
@@ -75,15 +75,12 @@ impl ObserverState for FrontendObserverNode {
             Info::HummockSnapshot(_) => {
                 self.handle_hummock_snapshot_notification(resp);
             }
-            Info::HummockVersionDeltas(_) => {
-                panic!("frontend node should not receive HummockVersionDeltas");
-            }
-            Info::MetaBackupManifestId(_) => {
-                panic!("frontend node should not receive MetaBackupManifestId");
-            }
             Info::SystemParams(p) => {
                 self.system_params_manager.try_set_params(p);
             }
+            _ => {
```

Comment:
How can we determine which type of error we received without removing the specific branches?

Reply:
I made this change to make it consistent with other observer impl.
I've reverted it.